.gitignore

```diff
@@ -0,0 +1,15 @@
+# Django #
+*.pyc
+db.sqlite3
+django_backend/tako_env
+__pycache__
+
+
+# Visual Studio Code # 
+.idea/
+.vscode/* 
+!.vscode/settings.json 
+!.vscode/tasks.json 
+!.vscode/launch.json 
+!.vscode/extensions.json 
+.history
```

Comment:
I think you could improve a little more this .gitignore file. Theres a website called [gitignoreio](https://www.toptal.com/developers/gitignore) that generate this file with pretty much everything that can and should be ignored in a project using Python.

Reply:
The .gitignore file was updated and commited to the branch.